packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div
+        :class="$style['c-link']"
+        data-test-id="link">
+        {{ copy.text }}
+    </div>
+</template>
+
+<script>
+import { globalisationServices } from '@justeat/f-services';
+import tenantConfigs from '../tenants';
+
+export default {
+    name: 'VLink',
+    components: {},
+    props: {
+        locale: {
+            type: String,
+            default: ''
+        }
+    },
+    data () {
+        const locale = globalisationServices.getLocale(tenantConfigs, this.locale, this.$i18n);
+        const localeConfig = tenantConfigs[locale];
+
+        return {
+            copy: { ...localeConfig }
+        };
+    }
+};
+</script>
+
+<style lang="scss" module>
+
+.c-link {
```

Comment:
Can change this in in a future PR, but should rename this to `o-link`

Reply:
I'll change it now so I remember 😂